.github/workflows/onPRClose.yml

```diff
@@ -50,13 +50,16 @@ jobs:
   destroy-deployed-branch:
     name: Call Azure Destroy Pipeline
     runs-on: ubuntu-latest
-    needs: get-details
     steps:
+    - name: Set new branch name value
```

Comment:
This naming is misleading since this is an destruction action, we're not really creating/setting a branch name.

We're trying to figure out what the deployment charname is so maybe change it for a better description?